applications/FluidDynamicsApplication/custom_utilities/cfd_utilities.cpp

```diff
@@ -0,0 +1,429 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya (https://github.com/sunethwarna)
+//
+
+// System includes
+#include <functional>
+#include <limits>
+
+// External includes
+
+// Project includes
+#include "utilities/math_utils.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/variable_utils.h"
+
+// Application includes
+#include "fluid_dynamics_application_variables.h"
+
+// Include base h
+#include "cfd_utilities.h"
+
+namespace Kratos
+{
+namespace CFDUtilities
```

Comment:
Why a separated namespace?

Reply:
Just to make sure that there aren't any conflicting names. Do you think it is not necessary?

Reply:
Well, I don't think it harms but we normally keep everything in the same namespace. If we want to start using them, I will consensus how to do so to be consistent among applications.

Reply:
Any comment about the namespace issues?

Reply:
Although it is ok for me to have everything under one namespace, I am in favor of using different namespaces :). In most of the utilities available in Kratos, we are used to have classes and public and/or static methods to nicely package them. I was thinking to do it using namespaces, so then these namespaces can be extended in different files to as it develops. But I have no clue on how to properly adhere to Kratos guide lines about use of namespaces :/ :|